### DISCUSSION TITLE
Source Code

### DISCUSSION THREAD

**Discussion by xSkeletor** [DB-db-dron/weatherDB#9] [General]

**Comment by DB-db-dron** [comment]
I am trying my best to shift the API to paid hosting this month to prevent this problem completely. I think the payment issue that I was facing is solved with PayPal.

It will be very helpful if you can tell how/where you are using the API, if not here, then through _Talk to Us_ section of the website.

**Reply by DB-db-dron** [reply]
@xSkeletor I recently tried out Render,. But as it does not support buildpacks like Heroku, the current instance of the API cannot be hosted there because it needs node.js and python both.

**Comment by shlomo20** [comment]
hi, 

@DB-db-dron 

I have to tell you this API has inspired me, I liked it so much that I couldn't just go back and use a different one, 
when it started to go down at the end of the month and wasn't reliable anymore for a production app, I went and created my own version (for private use only ), 

during the proses, I only got a lot more respect and appreciation for what it takes to maintain a project like this,

I now have a working version (modified with more endpoints for different data such as flight data), but... error handling in mine is almost non-existent, when requests increase google keeps rejecting the requests, it'll be nice if I can get a peek at how you're doing it....(and avoid my self some hours of hade scratching... although the experience wouldn't hurt ).

I am using this for a web app that allows the user to see and save data on cities and see an overview of 8 cities at once (for example current conditions ) then open a page to get all the details of that city.

I have been hosting my API on [vercel ](https://vercel.com/pricing)  (serverless functions) the free tear has been enough for my purposes, however, I am not getting the kind of traffic your site does.


Thanks again
